### PR TITLE
client-go: refactor: Fix styling issues

### PR DESCRIPTION
**Pull request by Volatus**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
It makes very minor changes to conform to coding style.
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/client-go/tools/cache/reflector.go [hunk 1]
```diff
@@ -258,7 +258,7 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 	options := metav1.ListOptions{ResourceVersion: r.relistResourceVersion()}
 
 	if err := func() error {
-		initTrace := trace.New("Reflector ListAndWatch", trace.Field{"name", r.name})
+		initTrace := trace.New("Reflector ListAndWatch", trace.Field{Key: "name", Value: r.name})
```
Comment (yliaog) [inline staging/src/k8s.io/client-go/tools/cache/reflector.go:261]: which coding style are you referring to for this change?

Comment (Volatus) [reply staging/src/k8s.io/client-go/tools/cache/reflector.go:261]: I guess it's not so much "styling" but it just makes the code more explicit.

**Comment by Volatus** [comment]
> /ok-to-test

The failing test is unrelated to this PR, no?

**Comment by yliaog** [comment]
/retest

probably the test failure is not related to the PR, but still it needs to pass.

**Comment by Volatus** [comment]
> 

do you think this could be merged now?

**Pull request merged**